stdlib/asyncio/taskgroups.pyi

```diff
@@ -0,0 +1,17 @@
+import sys
+from types import TracebackType
+from typing import Any, Coroutine, Generator, TypeVar
+
+from .tasks import Task
+
+_T = TypeVar("_T")
+
+if sys.version_info >= (3, 11):
```

Comment:
I believe this is unnecessary, because of the `VERSIONS` file. Other asyncio submodules don't have if statements like this.